addons/l10n_de/views/res_company_views.xml

```diff
@@ -3,7 +3,7 @@
         <record id="res_company_form_l10n_de" model="ir.ui.view">
             <field name="name">res.company.form</field>
             <field name="model">res.company</field>
-            <field name="inherit_id" ref="base.view_company_form"/>
+            <field name="inherit_id" ref="account.view_company_form"/>
```

Comment:
@sebalix  can you please update also l10n_de_reports?

Reply:
This one is part of `odoo/enterprise`, I'll open the PR there yes.